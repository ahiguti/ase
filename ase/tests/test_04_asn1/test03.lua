
local Test = ASE.CPP['ASETestCPP']
local arr = { 10, "foobar", 99.99999 }
local s = Test.ToDER(arr)
print(Test.ToHEX(s))
local v = Test.FromDER(s);
for i = 0, v.Length - 1 do
  print("v " .. i .. " " .. v[i])
end
