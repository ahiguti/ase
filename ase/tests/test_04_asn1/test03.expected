a11ca11a301809038001050c06666f6f626172090980d30c7fffeb074a77
v 0 10
v 1 foobar
v 2 99.99999
